back/config/schemas/style.schema.json

```diff
@@ -33,7 +33,8 @@
     },
     "signedOutHeaderOverlayColor": {
       "description": "The overlay color of the signed-out header",
-      "$ref": "#/definitions/color"
+      "$ref": "#/definitions/color",
+      "default": "#163A7D"
```

Comment:
@alexander-cit Tried to add default values here (for [this reason](https://citizenlab.atlassian.net/browse/CL2-6961)). But it doesn't work. Could you tell what I did wrong?

Reply:
As we discussed, `default` doesn't work with `style`.